src/main/kotlin/no/skatteetaten/aurora/cantus/service/DockerHttpClient.kt

```diff
@@ -122,7 +122,7 @@ class DockerHttpClient(
     }
 
     fun getImageTags(imageRepoCommand: ImageRepoCommand): ImageTagsResponseDto? = webClient
-        .request(imageRepoCommand, "/{imageGroup}/{imageName}/tags/list")
+        .request(imageRepoCommand, "{imageGroup}/{imageName}/tags/list")
```

Comment:
er dette en bevisst endring?

Reply:
Ja, vi la merke til at det ble logget med to skråstreker i metrikkene: `/v2//{imageGroup}/{imageName}/tags/list`

Reply:
topp, bare merge

Reply:
Vi så i metrikkene at det kom dobbel slash opp på dette endepunktet